src/wovnio/wovnphp/Store.php

```diff
@@ -207,17 +211,10 @@ private function updateSettings($updatedOptions)
         return $this->settings;
     }
 
-    private function ensureValidSupportedLanguages()
-    {
-        foreach ($this->settings['supported_langs'] as $index => $langCode) {
-            $this->settings['supported_langs'][$index] = $this->convertToOriginalCode($langCode);
-        }
-    }
-
-    private function ensureValidNoIndexLangs()
+    private function convertLangListToOriginalCodes(&$langListSetting)
```

Comment:
I want to avoid using `Passing by Reference` as much as possible. How about simply return original lang codes?